chingu-voyages/v52-tier3-team-39
# Save Form Data to Database

### To Test:

- on main branch, `git pull` to update
- `git checkout form-data-db`
- add `DATABASE_CONNECTION_STRING` and `MY_TEST_DB` variables to `.env` file in server application
- spin up client and server apps
- check server console to confirm that Mongo is connected
- navigate to "/form" in client app and submit a form
- on success, a new form document should be added to the `forms` collection in your database, and the app will redirect to "/success"

### Changes:

- add database configuration file that connects server to RayVolution Mongo cluster
- add environment variable config file
- add new auto-generated form fields: `dateCreated` (default `Date.now`) and `status` (default `"Pending"`)
- update `name` field validation
- save validated form data to `forms` collection in selected database
- move and rename files to improve clarity
- update README with env variable and route information
